castle/cms/__init__.py

```diff
@@ -1,4 +1,6 @@
+from zope.i18nmessageid import MessageFactory
 
+_ = MessageFactory('castle.cms')
```

Comment:
thanks for getting us on the path of translations!

Reply:
😄